airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -138,6 +156,11 @@ def _query_mysql(self):
         mysql = MySqlHook(mysql_conn_id=self.mysql_conn_id)
         conn = mysql.get_conn()
         cursor = conn.cursor()
+        # Ensure TIMESTAMP results are in UTC
+        tz_query = "SET time_zone = '+00:00'"
```

Comment:
Is it worth making this an argument to the op?

Reply:
I think this is resolved now :).